patches/chrome-browser-ui-views-tabs-tab_group_views.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/ui/views/tabs/tab_group_views.cc b/chrome/browser/ui/views/tabs/tab_group_views.cc
+index 56a7fb5e47d2a6be7f1db5c30310a905b9a09b78..1dbeace414cb10597b9cea5cd0228a233fc2a21e 100644
+--- a/chrome/browser/ui/views/tabs/tab_group_views.cc
++++ b/chrome/browser/ui/views/tabs/tab_group_views.cc
+@@ -172,6 +172,7 @@ TabGroupViews::GetLeadingTrailingGroupViews(
+   views::View* trailing_child = nullptr;
+   gfx::Rect trailing_child_widget_bounds;
+ 
++  BRAVE_GET_LEADING_TRAILING_GROUP_VIEWS
```

Comment:
this should also include class name: `BRAVE_TAB_GROUP_VIEWS_GET_LEADING_TRAILING_GROUP_VIEWS`.

Reply:
Yep. done.